src/brevitas/core/quant/float.py

```diff
@@ -35,10 +38,16 @@ def __init__(
         self.bit_width = StatelessBuffer(torch.tensor(float(bit_width), device=device, dtype=dtype))
```

Comment:
@Giuseppe5 if the sign bit is always there, this check also doesn't make sense anymore. If it is unsigned, the sign is just going to be 0, right? So one bit has to be reserved for the sign at all times anyways